### PR TITLE
fix shellcheck in cluster/gce/config-common.sh

### PR DESCRIPTION
**Pull request by beautytiger**
**What type of PR is this?**
/kind bug
/kind cleanup
/kind failing-test

**What this PR does / why we need it**:
Fix shellcheck failures in cluster/gce/config-common.sh
**Which issue(s) this PR fixes**:

Ref #72956

**Special notes for your reviewer**:
/cc @BenTheElder 
/assign @BenTheElder 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: cluster/gce/config-common.sh [hunk 1]
```diff
@@ -130,22 +130,22 @@ NODE_SCOPES="${NODE_SCOPES:-monitoring,logging-write,storage-ro}"
...
+export WINDOWS_KUBELET_CONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet-config.yaml"
 # Path for kubeconfig file on Windows nodes.
-WINDOWS_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.kubeconfig"
+export WINDOWS_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.kubeconfig"
 # Path for bootstrap kubeconfig file on Windows nodes.
-WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-kubeconfig"
+export WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-kubeconfig"
 # Path for kube-proxy kubeconfig file on Windows nodes.
-WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
+export WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
```
Comment (BenTheElder) [inline cluster/gce/config-common.sh]: are these read by anything? why do we need to export them?

Comment (BenTheElder) [reply cluster/gce/config-common.sh]: cc @yujuhong

File: cluster/gce/config-common.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@
 #   NUM_NODES
 #   NUM_WINDOWS_NODES
 function get-num-nodes {
-  echo "$((${NUM_NODES} + ${NUM_WINDOWS_NODES}))"
+  echo "$((NUM_NODES + NUM_WINDOWS_NODES))"
```
Comment (BenTheElder) [inline cluster/gce/config-common.sh]: 👍

File: cluster/gce/config-common.sh [hunk 3]
```diff
@@ -117,7 +117,7 @@ function get-cluster-ip-range {
 # $1: The number of max pods limitation.
 function get-alias-range-size() {
   for pow in {0..31}; do
-    if (( 1 << $pow >= $1 * 2 )); then
+    if (( 1 << pow >= $1 * 2 )); then
```
Comment (BenTheElder) [inline cluster/gce/config-common.sh]: 👍

**Review by BenTheElder — COMMENTED** [review]
/assign @yujuhong

File: cluster/gce/config-common.sh (hunk 1 above)
Comment (pjh) [reply cluster/gce/config-common.sh]: These are used in https://github.com/kubernetes/kubernetes/blob/3a501844219d06b6fd777a911175b823ab146d43/cluster/gce/util.sh#L1525.

**Comment by pjh** [comment]
I patched in this PR and successfully brought up a cluster with Windows nodes.

/lgtm

File: cluster/gce/config-common.sh (hunk 1 above)
Comment (beautytiger) [reply cluster/gce/config-common.sh]: Yes, thanks for your help. @pjh

Comment (BenTheElder) [reply cluster/gce/config-common.sh]: please add a comment somewhere in this file about this, so in the future if they are not used we can remove them

Comment (BenTheElder) [reply cluster/gce/config-common.sh]: gentle bump

**Comment by BenTheElder** [comment]
/assign
https://github.com/kubernetes/kubernetes/pull/82357#discussion_r324941736 is my last nit for approval, we should keep track of this

File: cluster/gce/config-common.sh (hunk 1 above)
Comment (beautytiger) [reply cluster/gce/config-common.sh]: Got it, sorry for late reply. Please check again.

**Pull request merged**